src/helpers/timeout.js

```diff
@@ -193,6 +214,36 @@ var Timeout = (function() {
       clearInterval(waitForDOMIntervalId);
     }
   }
+  function getModalElement() {
+    return document.querySelector("#timeoutModal");
+  }
+  function openModal() {
+    var modalElement = getModalElement();
+    if (!modalElement) return;
+    modalElement.classList.add("modal-open");
+    modalElement.classList.remove("modal-close");
+    document.querySelector("body").classList.add("fixed");
+    document.querySelector("body").classList.add("ReactModal__Body--open");
+  }
+  function initTimeoutModal() {
+    if (getModalElement()) return;
+    var modalElement = document.createElement("div");
+    modalElement.setAttribute("id", "timeoutModal");
+    modalElement.classList.add("modal-close");
+    modalElement.innerHTML = `<div class="ReactModal__Overlay ReactModal__Overlay--after-open overlay">
+      <div class="ReactModal__Content ReactModal__Content--after-open modal small" tabindex="-1" role="dialog" aria-label="Timeout notice">
+          <div class="info-modal">
+              <div class="info-modal__header">Session Timeout Notice</div>
+              <div class="info-modal__content text-bold">
+                <h2 class="error">Your session is about to time out.</h2>
+                <p>One moment while we redirect you back to Patient Search...</p>
```

Comment:
```suggestion
                <p>One moment while we redirect you back...</p>
```

We may want to make this more generic, in case we're launching out of an EHR like Epic, but I'm not sure how to phrase it without being too vague...